sw/device/silicon_creator/keys/README.md

```diff
@@ -0,0 +1,98 @@
+This directory contains ASN1 DER encoded development keys for signing ROM\_EXT
+images. Until we have a more ergonomic tool for working with keys and signatures,
+below snippets can be used for various operations.
```

Comment:
I agree that CLI is not great, but I don't think we will need to update these keys again soon, at least until we have this functionality in `opentitantool`, because we have keys with both exponents. I thought it would be better to focus our efforts there.